core/modules/compare/components/Compare.ts

```diff
@@ -0,0 +1,47 @@
+import i18n from '@vue-storefront/i18n'
+import Product from '@vue-storefront/store/types/product/Product'
+
+export const Compare = {
+  name: 'Compare',
+  props: {
+    title: {
+      type: String,
+      required: true
+    }
+  },
+  metaInfo () {
+    return {
```

Comment:
should be moved to theme. If this is meant to be a component instead of page (which is correct) then it shouldn't interact with meta.

Reply:
moved to theme.